webapp/channels/babel.config.js

```diff
@@ -5,14 +5,12 @@ const config = {
     presets: [
         ['@babel/preset-env', {
             targets: {
-                chrome: 66,
-                firefox: 60,
-                edge: 42,
-                safari: 12,
+                chrome: 110,
```

Comment:
Also, I updated `preset-env` to match [our latest software requirements](https://docs.mattermost.com/install/software-hardware-requirements.html#pc-web), so it should be only using adding polyfills for things that those browsers don't support